Postman Test Suite/Ed-Fi ODS-API Integration Test Suite.postman_collection.json

```diff
@@ -175,12 +175,10 @@
 							]
 						},
 						"url": {
-							"raw": "http://localhost:8765/oauth/token",
-							"protocol": "http",
+							"raw": "{{ApiBaseUrl}}/oauth/token",
```

Comment:
This should not be hard-coded to the integration test harness port number. It should be driven by the Postman environment variables.